### DISCUSSION TITLE
Can I use Inline Scripts Inline scripts, but at the same time let the JS code inside get compiled?

### DISCUSSION THREAD

**Discussion by zerofront** [vercel/next.js#39019] [Help]
Can I use Inline Scripts Inline scripts, but at the same time let the JS code inside get compiled?

<img width="733" alt="image" src="https://user-images.githubusercontent.com/19327366/180938346-2e698a6a-8f12-404a-8689-44479631edb7.png">

Now the inline script component is full of strings, can I import the code of the project?

**Comment by valcosmos** [comment]
yes, maybe u can try this.

hope it can helps u.

```tsx

<Script src="https://example.com/non-existant-script.js" type="module" />

```

**Reply by zerofront** [reply]
> yes, maybe u can try this.
> 
> hope it can helps u.
> 
> ```tsx
> <Script src="https://example.com/non-existant-script.js" type="module" />
> ```

can I import js code from Same project？ 
I don't understand https://example.com/non-existant-script.js

**Reply by valcosmos** [reply]
yes, u can replace the link in src with the js link you want to import.

"https://example.com/non-existant-script.js" it is just a example link.
